components/MeetingDetail/MeetingDetail.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import DogName from "../DogName/DogName";
+import ImageWithText from "../ImageWithText/ImageWithText";
+
+export default function MeetingDetail({ data }) {
+  const { location, date, time, excluded, furtherInfo, attending } = data;
+  return (
+    <StyledArticle>
+      <h2>{location}</h2>
+      <ImageWithText
+        image="calendar-icon.svg"
+        altText="Kalender Icon"
+        text={date}
+      />
+      <ImageWithText image="clock-icon.svg" altText="Uhr Icon" text={time} />
+      {excluded ? (
+        <ImageWithText
+          data-testid="excluded"
+          image="stop-icon.svg"
+          altText="Stop Icon"
+          text={excluded}
+        />
+      ) : null}
+      <StyledInfobox>
+        {furtherInfo ? (
+          <div data-testid="info">
+            <label htmlFor="weitereInfos">Weitere Infos:</label>
```

Comment:
A `label` is normally used for `input` elements, but you don't have one here, right?

Reply:
True. I was unsure about this. Changed it. Thanks!